packages/block-library/src/media-text/editor.scss

```diff
@@ -50,6 +50,10 @@ figure.block-library-media-text__media-container {
 	width: 100%;
 }
 
+.wp-block-media-text .block-library-media-text__media-container .components-placeholder__preview img {
```

Comment:
This is a pretty long selector, were there specificity issues? If so, it'd be great to add a comment mentioning it.